api/sql/schema/750$transfer.report.settlementDetails.sql

```diff
@@ -132,16 +132,18 @@ BEGIN TRY
         JOIN
             [card].[card] c ON c.cardId = v.cardId
         JOIN
+            [card].[type] ct ON c.typeId = ct.typeId
```

Comment:
The many changes in this SP could have been avoided if we had a view [card].vCard